### ISSUE TITLE
Stored properties cannot be marked unavailable with '@available'

### ISSUE THREAD

**Issue by lexuanquynh** [MagicTimerFW/MagicTimer#23]
Build error in Xcode 15. Can you help me update the this repository?
<img width="1112" alt="Screenshot 2023-09-25 at 13 31 25" src="https://github.com/MagicTimerFW/MagicTimer/assets/17588791/e09a413d-ff99-4d77-8f20-672bce77246f">

**Comment by ShravaniNovo** [comment]
Build error in Xcode 15. Can you help me update the this repository?  i am also having same issue can you please give alternate solution to it.

**Comment by lexuanquynh** [comment]
@ShravaniNovo 
Step 1: fork this repo to your repo.
Step 2: Fix in your repo.
Step 3: Change Podfile link to your repo.
Something like that:
```
pod "MagicTimer", :git => 'https://github.com/<Your repo>/MagicTimer.git', :branch => 'master'
```

**Comment by alexcho617** [comment]
Having the same issue here
<img width="787" alt="image" src="https://github.com/MagicTimerFW/MagicTimer/assets/38528052/56482649-fb21-4322-8794-062c0213e7ab">

**Comment by sadeghgoo** [comment]
It has been fixed in 1.0.5 version.

**Comment by PriyankaNovo** [comment]
Unable to find a specification for `MagicTimerCore` depended upon by `MagicTimer` throwing this error while updating pod

**Comment by ShravaniNovo** [comment]
> It has been fixed in 1.0.5 version.

Unable to find a specification for `MagicTimerCore` depended upon by `MagicTimer` throwing this error while updating pod.

**Comment by alexcho617** [comment]
@sadeghgoo Thank you! love your timer :)

**Comment by harooniqbal1993** [comment]
> The Problem with Cocoapods has been fixed: pod 'MagicTimer', '~> 1.0.6'

@sadeghgoo, I was using MagicTimerView with UIKit in my old project, is it deprecated in v1.0.6?
